File: src/main/java/com/thegame/ui/controller/UserController.java
package com.thegame.ui.controller;

import com.thegame.business.service.UserService;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
public class UserController {

    private final UserService userService;

    public UserController(UserService userService) {
        this.userService = userService;
    }

    @RequestMapping(value = "users.html")
    public String getUsers(Model model) {
        model.addAttribute("users", userService.getAllUsers());
        return "users";
    }
}

File: src/main/java/com/thegame/ui/controller/UserController.kt
package com.thegame.ui.controller

import com.thegame.business.service.UserService
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.RequestMapping

@Controller
class UserController(private  val userService: UserService) {

    @RequestMapping(value = ["users.html"])
    fun getUsers(model: Model): String {
        model.addAttribute("users", userService.getAllUsers())
        return "users"
    }
}